project/webapp/utils/Constants.js

```diff
@@ -9,5 +9,8 @@ sap.ui.define([], function () {
     APP_MODEL_NAME: "appModel",
     EDIT_MODEL_NAME: "editModel",
     FILTER_BAR_MODEL_NAME: "filterBarModel",
+    SUPPLIERS_MODEL_NAME: "suppliersModel",
+    COUNTRIES_MODEL_NAME: "countriesModel",
+    API_KEY: "WlFWdU45MlpsQXR6endFWGhjMmR1cHg0TGlLaTA5QzJZVXBRd0MyTQ=="
```

Comment:
I understand that you did it to make it easier testing for me, but lets move it .env file, you can provide me the key privately in messenger)